Add Metrics module

mijicd opened the pull request.

mijicd committed:
Add Metrics module

mgobec commented on projects/anomaly-detection-stack/inventory/group_vars/smart-cassandra.yml:
```diff
@@ -37,6 +37,9 @@ cassandra_diagnostics_binaries:
   - "{{ cassandra_diagnostics_url_connector }}"
   - "{{ cassandra_diagnostics_url_reporter_riemann }}"
 
+jmx_host: 127.0.0.1
```
Prefix these variables so it makes sense

mijicd replied:
Fixed in [64de10e](https://github.com/smartcat-labs/smartcat-ops/pull/56/commits/64de10e0c644ac0bff9af7ca1993091d10eefac0)

mijicd committed:
Rename JMX variables

Variables 'jmx_host' and 'jmx_port' are prepended with
'cassandra_diagnostics'. Documentation is updated to reflect the
cassandra diagnostics version upgrade.
